gui/templates/system/system_info.html

```diff
@@ -93,7 +93,7 @@ <h1>{% trans "System Information" %}</h1>
         <th>{% trans "Load Average" %}</th>
         <td class="descr">{{ local.loadavg }}</td>
     </tr>
-    {% if not is_freenas %}
+    {% if system_manufacturer == 'iXsystems' %}
```

Comment:
I think it should be ```local.system_manufacturer```?

Reply:
Yes; And I have a correct version on my testing system. It's a typo.

Reply:
I would suggest mounting your git repo inside your testing system using NFS, that will prevent issues like that ; )